templates/pages/category.html

```diff
@@ -18,7 +18,11 @@
 
 {{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
 {{#if category.image}}
-    <img class="lazyload" data-sizes="auto" src="{{cdn 'img/loading.svg'}}" data-src="{{getImage category.image 'zoom_size'}}">
+    {{> components/common/responsive-img
+        image=category.image
+        fallback_size='1280w'
```

Comment:
Shouldn't this be defined by `zoom_size`?

Reply:
Yep, I'll make these updates. I think my thinking at the time is that it seemed like we were sort of overloading those sizes (based on what they say they do in Theme Editor), but I can certainly put them back for consistency. I'll do that now and resolve these.